tests/models/test_taskinstance.py

```diff
@@ -3192,6 +3192,22 @@ def raise_skip_exception():
         assert State.SKIPPED == ti.state
         assert callback_function.called
 
+    def test__run_raw_task_decorated_with_sentry_enrich_errors(self, create_task_instance):
+        """
+        Test TaskInstance._run_raw_task is decorated with encountering error.
+        """
+        ti = create_task_instance()
+        ti._run_raw_task()
```

Comment:
How are these new tests checking that the methods are decorated?

Reply:
Yep, you're right. This might need some modification. The original attempt was to ensure running these methods does not encounter any error, which is the root cause of https://github.com/apache/airflow/issues/36957

Reply:
I tired to make sure that function in the decorator is called in the latest change